### DISCUSSION TITLE
Can exported Google Keep notes content be imported into memos?

### DISCUSSION THREAD

**Discussion by jon-f-novabackup** [usememos/memos#1705] [General]
Can exported Google Keep notes content be imported into memos? I realize Google Keep itself doesn't have an export notes function, visible in the Google Keep UI directly, however I do see that there is a way to export all of your Google Keep notes content (notes and image attachments) to multiple formats including .HTML and .JSON, using the https://takeout.google.com export Google Keep notes function, as seen below:

![image](https://github.com/usememos/memos/assets/50121594/19e9740a-58a3-46af-a597-91ecd668b85b)

The "More info" button seen in the top left of the above screenshot, shows the different Keep formats that you can export your data in, including HTML and JSON formats:

![image](https://github.com/usememos/memos/assets/50121594/a7061a86-7674-405b-bfa3-5f46b512836c)

I was just hoping to save some time manually re-creating all of my stored notes in Google Keep, to get at least the notes themselves (I can always add the images in later) imported as memos note entries, if I don't have to. I don't believe this is possible in the latest 0.13.0 version of memos but I'm just checking. Thanks for the great app!

**Comment by Zeng1998** [comment]
Hey, I suggest you follow [this repo](https://github.com/hyoban/kirika) from @hyoban and make feature requests if you need it.

The import and export of Memos will be implemented in this repo.

**Comment by hyoban** [comment]
You can see the progress here. 
https://github.com/hyoban/kirika/issues/3

**Reply by jon-f-novabackup** [reply]
Thanks @hyoban for trying implement that convert feature into https://github.com/hyoban/kirika.
